dynamic-rapidpro-workspace-settings/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "update-rapidpro-workspace-settings",
+  "version": "1.0.0",
+  "description": "dynamic-rapidpro-workspace Shared GitHub Action",
+  "main": "index.js",
+  "scripts": {
+    "build": "ncc build index.js --license licenses.txt",
+    "test": "mocha test/*.spec.js --timeout 10000 --slow 500"
+  },
+  "author": "Medic Inc.",
+  "license": "ISC",
```

Comment:
Medic code is released under AGPL (AGPL-3.0-only)

Reply:
Most repos use `ISC`. Updated to `AGPL-3.0-only`.